src/api/wallet.js

```diff
@@ -27,7 +27,7 @@ const walletApi = {
    */
   getAddressHistory(addresses, resolve) {
     const data = {addresses};
-    return createRequestInstance(resolve).get('thin_wallet/address_history', {'params': data}).then((res) => {
+    return createRequestInstance(resolve, 40000).get('thin_wallet/address_history', {'params': data}).then((res) => {
```

Comment:
I don't think we should increase the time limit.

Reply:
Unless we get the pagination PR (#47) as well, we need to increase it.

Reply:
Ok. We can change it, but we need to undo it after the PR #47 is merged. So, I left a comment in PR #47 to reduce this time limit.